src/pathwayvis/components/map/map.component.ts

```diff
@@ -1,21 +1,21 @@
 import * as escher from '@dd-decaf/escher';
 import * as d3 from 'd3';
-import {event as currentEvent} from 'd3';
+import { event as currentEvent } from 'd3';
 import * as _ from 'lodash';
+import * as tinier from 'tinier';
 
 import { APIService } from '../../services/api';
 import { WSService } from '../../services/ws';
 import { ActionsService } from '../../services/actions/actions.service';
 import { SharedService } from '../../services/shared.service';
-
 import * as types from '../../types';
-
 import './views/map.component.scss';
 import * as template from './views/map.component.html';
 import { ToastService } from "../../services/toastservice";
 import { MapOptionService } from "../../services/mapoption.service";
 import { ObjectType } from "../../types";
 import * as utils from "../../utils";
+import * as escherTooltip from "../tooltip/escherTooltip";
```

Comment:
why don't you import `{ Tooltip }`? And then you could just pass `Tooltip` here https://github.com/DD-DeCaF/metabolica-ui-map/pull/141/files#diff-62c06d0a48047fe88322f20be6e856e2R364